site/ja/js/tutorials/index.md

```diff
@@ -0,0 +1,51 @@
+# はじめに
+
+TensorFlow.jsは ブラウザとNode.js内で 機械学習モデルの訓練とデプロイを行うための JavaScriptライブラリです。
```

Comment:
It better to insert white space between alphabetical word and Hiragana (or Katakana).
We are currently discussing on translation guideline for Japanese.
How do you think about it?

Reply:
The space between an alphabetic word and a Japanese word should be the issue of style. I don't think it's good idea to insert white space between them by hand.